WordPress/gutenberg
# Components: Avoid displaying tooltip on focus from mousedown

This pull request seeks to resolve an issue where a button with an associated tooltip will briefly flash the tooltip when clicked. The underlying issue here is that a tooltip is expected to be shown (immediately) when a button is focused. Previously, this was not disambiguated from focus which results from clicking to interact with the button, where the tooltip is not expected to be shown.

Before|After
---|---
![tooltip-before](https://user-images.githubusercontent.com/1779930/62081419-94ffcb00-b220-11e9-9bf8-08015e296c1c.gif)|![tooltip](https://user-images.githubusercontent.com/1779930/62081422-96c98e80-b220-11e9-9bfd-cbe17c6a957c.gif)

**Testing instructions:**

Verify there are no regressions in the display of the tooltips being shown upon focusing the button.

Verify that the tooltip is not shown when clicking on a button with an associated tooltip (e.g. inserter, block movers, etc).

_Needs Accessibility Feedback:_ I'd appreciate any advice on whether I've overlooked any use cases with the assumption that focus-via-click should be ignored.
